### PR TITLE
(DOCSP-18928): Compass logs

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
### Description

This was a feature introduced a while ago that we missed documenting. As noted in [DOCSP-21469](https://jira.mongodb.org/browse/DOCSP-21469), in addition to creating a main page for logging, we also want to update the Troubleshooting, FAQ, and Release Notes for this feature.

### Staging Links

- [Retrieve Compass Logs](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-18928/troubleshooting/logs/)
- [New FAQ Item](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-18928/faq/#where-can-i-find-compass-logs-)
- [Connection Troubleshooting](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-18928/troubleshooting/connection-errors/#check-the-compass-logs)
- [1.29.4 Release Notes](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-18928/release-notes/#compass-1.29.4)

### JIRA

- [DOCSP-18928](https://jira.mongodb.org/browse/DOCSP-18928)
- [DOCSP-21469](https://jira.mongodb.org/browse/DOCSP-21469)

### CODE REVIEW

File: source/faq.txt [hunk 1]
```diff
@@ -51,6 +51,15 @@ data in the operating system's credentials API. For more information on
 how Keytar operates and the specific APIs it accesses, refer to the
 `Keytar Github documentation <https://github.com/atom/node-keytar>`_.
 
+.. _compass-faq-logs:
+
+Where can I find Compass logs?
```
Comment (jmd-mongo) [inline source/faq.txt:56]: [comment]

I think this section could benefit from being more direct. The first two sentences describe what logs *are*, but not where to find them. The final sentence reads more like a "see also" in its current form than a clear imperative. 

I recommend making the link language more explicit.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @jeff-allen-mongo! I've left a few comments and questions for you.

Thanks!
Joe

File: source/troubleshooting/logs.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 20
+
+   * - Operating System
+     - Directory
+   * - macOS
+     - ``~/.mongodb/compass/<LogID>_log``
```
[the remark this answers is not in the document]
Comment (mmarcon) [reply source/troubleshooting/logs.txt:60]: I checked both Windows and macOS and the log files do, indeed, have the `.gz` extension.

**Review by jeff-allen-mongo — COMMENTED** [review]
@jmd-mongo, thanks for your feedback! Would you mind taking another look at this one?

Jeff

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates, @jeff-allen-mongo! This one LGTM.

Thanks,
Joe

**Pull request merged**